bigtop_toolchain/bin/puppetize.sh

```diff
@@ -40,7 +40,9 @@ case ${ID}-${VERSION_ID} in
         ;;
     centos-7*)
         rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-        yum -y install hostname curl sudo unzip wget puppet puppetlabs-stdlib
+        yum updateinfo
+        yum -y install hostname curl sudo unzip wget puppet
+        puppet module install puppetlabs-stdlib --version 4.12.0
```

Comment:
Mind to add a comment here to link to BIGTOP-3088? 
This tells future developer that this line is for special case and can be removed if obstacle removed.

Reply:
Thanks. The suggestion reaaly helps. I used to run into similar case without knowing the code block's context.
Will update soon.